token/client/src/token.rs

```diff
@@ -2429,4 +2429,27 @@ where
         )
         .await
     }
+
+    pub async fn withdraw_excess_lamports<S: Signers>(
+        &self,
+        source: &Pubkey,
+        authority: &Pubkey,
+        destination: &Pubkey,
```

Comment:
nit for consistency with `transfer`:
```suggestion
        source: &Pubkey,
        destination: &Pubkey,
        authority: &Pubkey,
```